datamol/conformers/_conformers.py

```diff
@@ -170,8 +175,11 @@ def generate(
     # Minimize energy
     if minimize_energy:
 
-        # Minimize conformer's energy using UFF
-        results = rdForceFieldHelpers.UFFOptimizeMoleculeConfs(mol, maxIters=energy_iterations)
+        # Minimize conformer's energy using MMFF
```

Comment:
@matudor, please allow using the previous UFF, and don't replace the default without an option to retain previous behaviour.